recipes/mp2d/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "mp2d" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Chandemonium/MP2D/archive/c6c7f866d4c1913ff38854b2c38450184888798a.zip
```

Comment:
Is there a stable tag we can point to? If not I'd rather be explicit that this is a commit and name it a `dev` release, fowlling cfep-05. BTW, it seems that @loriab is a contributor of this library, maybe Lori can help us with a release?

Reply:
No need to fork it unless you wish to maintain it. But I'd rather call it "dev" to mark it as not stable and follow cep-05. unless you have arguments that this is indeed quite stable and should be on main.

Reply:
Ideally, it would be best to have it on `main` as I think that package is stable enough. But I'll let @loriab makes the call here.